simtools/model/mirrors.py

```diff
@@ -103,22 +132,30 @@ def _read_mirror_list_from_sim_telarray(self):
                 "shape_type",
                 "mirror_z",
                 "sep",
-                "mirror_id",
+                "mirror_panel_id",
             ],
+            units=["cm", "cm", "cm", "cm", None, "cm", None, None],
         )
 
         self.shape_type = self.mirror_table["shape_type"][0]
-        self.mirror_diameter = self.mirror_table["mirror_diameter"][0]
+        self.mirror_diameter = u.Quantity(
+            self.mirror_table["mirror_diameter"][0], self.mirror_table["mirror_diameter"].unit
+        )
         self.number_of_mirrors = len(self.mirror_table["focal_length"])
 
         self._logger.debug(f"Mirror shape_type = {self.shape_type}")
         self._logger.debug(f"Mirror diameter = {self.mirror_diameter}")
         self._logger.debug(f"Number of Mirrors = {self.number_of_mirrors}")
 
-        if self.number_of_mirrors == 0:
-            msg = "Problem reading mirror list file"
-            self._logger.error(msg)
-            raise InvalidMirrorListFile()
+    def get_mirror_table(self):
```

Comment:
You should use the property mechanism of python instead of this classic getter (you can find lots of examples online).

Reply:
I thought we said you will remove this function. It's not a show stopper for this PR though.